Tools/refcounts/lint.py

```diff
@@ -72,7 +74,7 @@ def is_c_parameter_name(name: str) -> bool:
     return name == C_ELLIPSIS or name.isidentifier()
 
 
-class RefEffect(enum.Enum):
+class Effect(enum.Enum):
     """The reference count effect of a parameter or a return value."""
 
     UNKNOWN = enum.auto()
```

Comment:
nit: I think `auto()` just sets the value to an integer, but here `None` seems more appropriate given all other variants have `str` values

```suggestion
    UNKNOWN = None
```